### PR TITLE
Fix the MaxFileSizeInKilobytes change not effective

### PR DESCRIPTION
**Pull request by shaopeng-gh**
1. Fix the MaxFileSizeInKilobytes change not effective
2. Update baseline

### CODE REVIEW

File: src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs [hunk 1]
```diff
@@ -59,14 +59,14 @@ public override BinaryAnalyzerContext InitializeGlobalContextFromOptions(Analyze
                 context.Logger = aggregatingLogger;
             }
 
-            base.InitializeGlobalContextFromOptions(options, ref context);
-
             // We override the driver framework size default to be as large as
             // possible Binaries and (in particular) their PDBs can be large.
             context.MaxFileSizeInKilobytes = options.MaxFileSizeInKilobytes != null
                 ? options.MaxFileSizeInKilobytes.Value
                 : long.MaxValue;
```
Comment (shaopeng-gh) [inline src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs:66]: This code was too late, the files were already skipped

File: src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/clangcl.pe.cpp.codeview.exe.sarif [hunk 2]
```diff
@@ -3,60 +3,995 @@
...
+            "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/clangcl.pe.cpp.codeview.exe",
+            "index": 0
           },
           "hashes": {
-            "md5": "F983F893F6C5181CFA6574B520143F07",
-            "sha-1": "AA8AC9104D7917243AE052942C5F40BA929415AF",
-            "sha-256": "AC5BDD260432A32F62756ED8CF6E4297DAFFF0E6BA01C9522CC956C6D894B5C2"
+            "md5": "3AA0BC84045DCF91EF2B136E0B741200",
+            "sha-1": "C5EBC1B2D5BCCF44D69A61C437A7701958B30E07",
+            "sha-256": "918D85AEEDEACCC6D9E9E64C851560900BDA0CB554148645FF0295E403A21979"
```
Comment (shaopeng-gh) [inline src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/clangcl.pe.cpp.codeview.exe.sarif:994]: note the hashes change. 
This was a bug from SDK.

my comment here:
[https://github.com/microsoft/binskim/pull/853#discussion_r1164644217](https://github.com/microsoft/binskim/pull/853#discussion_r1164644217)

**Review by suvamM — APPROVED** [review]

**Review by suvamM — APPROVED** [review]

**Review by schlaman-ms — APPROVED** [review]
:shipit:

File: src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs (hunk 1 above)
Comment (michaelcfanning) [reply src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs:66]: Thanks for this fix!

**Pull request merged**